source/glossary.rst

```diff
@@ -20,10 +20,11 @@ Glossary of Terms
 
    crossfader
      The crossfader is a slider that determines how much each deck of audio
-     contributes to the :term:`master output`.
+     contributes to the :term:`main output`.
 
+   main output
    master output
```

Comment:
maybe emphasize that the term "master output" has been used historically, but is being phased out so people will know which one to use?

Reply:
Can you suggest a wording? I'm a bit uncreative today.

Reply:
Maybe add an extra sentence at the end of the description like this:
> Please note: the term "master output" has been used historically but is in the [process of being phased out](https://mixxx.org/news/2020-06-29-black-lives-matter/). Prefer using "main output".

I'm not too good at writing either so feel free to make changes as you see fit